package.json

```diff
@@ -10,7 +10,9 @@
     "lint:js": "eslint --ext .js,.vue --ignore-path .gitignore .",
     "lint:style": "stylelint **/*.{vue,css} --ignore-path .gitignore",
     "lint": "npm run lint:js && npm run lint:style",
-    "test": "jest"
+    "test": "jest",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
https://github.com/hikariumeno/nuxt.js-practice/blob/eb9c4bd6169e78601a786007571fb6d6687b2a49/.github/workflows/node.js.yml#L28

please insert `yarn build-storybook` before `test` after `npm run build` :)

Reply:
oops, `npm run build-storybook`

Reply:
thank you, I fixed in this commit d02052ccc1f9ed896b97150e26bcdc969979776c